StyleCop.Analyzers/StyleCop.Analyzers.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -23,10 +23,7 @@ public abstract partial class DiagnosticVerifier
         /// <returns>
         /// A new instance of the C# analyzer being tested.
         /// </returns>
-        protected virtual DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
-        {
-            return null;
-        }
+        protected abstract DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer();
```

Comment:
A very good change, but I think it would be better to make a separate pull request for it.